lib/ueberauth/strategy/google.ex

```diff
@@ -128,6 +128,8 @@ defmodule Ueberauth.Strategy.Google do
         set_errors!(conn, [error("token", "unauthorized")])
       {:ok, %OAuth2.Response{status_code: status_code, body: user}} when status_code in 200..399 ->
         put_private(conn, :google_user, user)
+      {:error, %OAuth2.Response{status_code: status_code}} ->
+        set_errors!(conn, [error("OAuth2", status_code)])
```

Comment:
Should we perhaps grab the body and use that as the error reason in place of the status code?

Reply:
In my case, the body was an empty map, so probably not

Reply:
See my comment here https://github.com/ueberauth/ueberauth_google/issues/39#issuecomment-345282172

Reply:
Okay.  Let's go with this for now, someone can always open a PR later 👍